contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -90,9 +85,12 @@ contract LidoOracle is ILidoOracle, AragonApp {
     /// @dev storage for all gathered from reports data
     mapping(uint256 => EpochData) private gatheredEpochData;
     /// @dev historic data about 2 last completed reports
-    bytes32 internal constant COMPLETED_REPORT_ID_LAST = bytes32(0);
-    bytes32 internal constant COMPLETED_REPORT_ID_PREV = bytes32(1);
-    mapping(bytes32 => CompletedReport) private lastCompletedReports;
+    bytes32 internal constant LAST_COMPLETED_TOTAL_POOLED_ETHER_POSITION =
+        keccak256("lido.LidoOracle.lastCompletedTotalPooledEther");
+    bytes32 internal constant PREV_COMPLETED_TOTAL_POOLED_ETHER_POSITION =
```

Comment:
It's not prev/last completed total pooled ether - it's pre-report and post-report, I feel it's better to make that disticntion explicit.

Reply:
Renamed to `pre/postTotalPooledEther`. Please feel free to tell me if you meant anything different.